webpack.config.js

```diff
@@ -253,6 +259,38 @@ module.exports = {
       ],
       ServiceWorker: {navigateFallbackURL: '/'},
     }),
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: JSON.stringify(process.env.NODE_ENV || 'development'),
```

Comment:
Do we need the `stringify` here?

Reply:
Yup, I think we do:

> Note that because the plugin does a direct text replacement, the value given to it must include **actual quotes** inside of the string itself. Typically, this is done either with either alternate quotes, such as `'"production"'`, or by using `JSON.stringify('production')`.

From https://webpack.js.org/plugins/define-plugin/

Reply:
well then, in my face.